controllers/nodehealthcheck_controller.go

```diff
@@ -226,7 +227,7 @@ func (r *NodeHealthCheckReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 			Reason:  remediationv1alpha1.ConditionReasonEnabled,
 			Message: enabledMessage,
 		})
-		r.Recorder.Eventf(nhc, v1.EventTypeNormal, utils.EventReasonEnabled, enabledMessage)
+		commonevents.NormalEvent(r.Recorder, nhc, utils.EventReasonEnabled, enabledMessage)
```

Comment:
I think the commonevnts are mostly intended for remediators as it has the [`[remediation]`](https://github.com/medik8s/common/blob/main/pkg/events/events.go#L12C7-L12C16) prefix.
We might want to make it more general for any events not from remediators or related to remediation before using it widely as the PR suggests.
It can be done by using another wrapper function in commonevnts for remediator events and other events

Reply:
The prefix is `remediation`, not `remediators`. And NHC also is about `remdiation`, isn't it? So I don't see an issue here...

Reply:
the idea was to filter events for `[remediation]` and see everything related to it.

Reply:
I agree, I was just not sure whether events Enabled and Disabled are also relevant for the remediation. But they are and as you mentioned NHC is about remediation. So there is no issue here.